README.md

```diff
@@ -27,18 +27,23 @@ Bootstrap-cfn uses [fabric](http://www.fabfile.org/), so if your ```$CWD``` is t
 ```
 fab application:courtfinder aws:prod environment:dev config:/path/to/courtfinder-dev.yaml cfn_create
 ```
-
-If your ```CWD``` is anywhere else, you need to pass in a path to particular fabric file:
-
+This will create a stack with a name like `courtfinder-a1e8-prod`. To delete this stack, run:
 ```
-fab -f /path/to/bootstrap-cfn/fabfile.py application:courtfinder aws:prod environment:dev config:/path/to/courtfinder-dev.yaml cfn_create
+fab application:courtfinder-a1e8 aws:prod environment:dev config:/path/to/courtfinder-dev.yaml cfn_delete
 ```
+and `cfn_delete:force` to skip the prompt.
```

Comment:
It's worth knowing that this actually ends up with calling `cfn_delete(force="force")`.

I'm not sure it matters in this case but if we want it can be invoked like `fab cfn_delete:force=True`